internal/cli/atlas/deployments/setup.go

```diff
@@ -572,7 +586,7 @@ func (opts *SetupOpts) Run(ctx context.Context) error {
 		return err
 	}
 
-	cs := fmt.Sprintf("mongodb://localhost:%d", opts.Port)
+	cs := fmt.Sprintf("mongodb://localhost:%d/?directConnection=true", opts.Port)
```

Comment:
I wonder if we should do `?replicaSet=rs-local` instead

Reply:
from https://www.mongodb.com/docs/drivers/go/current/fundamentals/connection/:
```
When connecting to a replica set, the driver takes the following actions by default:
Discovers all replica set members when given the address of any one member.
Dispatches operations to the appropriate member, such as instructions to write against the primary.
```
```
To force operations on the host designated in the connection URI, specify the directConnection option
```
The discovery process will return the podman hostname (e.g. mongod-local636), and then the connection will fail with:
`getaddrinfo ENOTFOUND mongod-local636`